src/cli/commands/outdated.js

```diff
@@ -44,6 +44,9 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     });
 
     reporter.table(['Package', 'Current', 'Wanted', 'Latest', 'Package Type', 'URL'], body);
+    process.on('exit', () =>
+      process.exitCode = 1
```

Comment:
That looks like a hack.
We prefer to have a single exit code in cli/index.js that will be set code 1 if the command returned a rejected promise